scripts/notice-check.py

```diff
@@ -25,7 +25,11 @@
     ".cpp": [SLASH_PREFIXED],
     ".h": HEADERS_WITH_PRAGMAS,
     ".hpp": HEADERS_WITH_PRAGMAS,
-    ".py": [HASH_PREFIXED],
+    ".py": [
+        HASH_PREFIXED,
+        # May have a shebang before the license
+        "#!/usr/bin/env python3" + os.linesep + HASH_PREFIXED,
```

Comment:
Not something we commonly do, because our scripts are otherwise largely packaged, but useful for standalone scripts like tlc.py.